src/Jyoti_Misc/Assignment69.java

```diff
@@ -0,0 +1,50 @@
+package Jyoti_Misc;
+
+import java.util.Scanner;
+
+public class Assignment69 {
+
+	int daysIndex(int index) {
+
+		switch (index) {
+		case 1:
+			System.out.println("Weekday :" + index + " and Day is Monday");
+			break;
+		case 2:
+			System.out.println("Weekday :" + index + " and Day is Tuesday");
+			break;
+		case 3:
+			System.out.println("Weekday :" + index + " and Day is Wednesday");
+			break;
+		case 4:
+			System.out.println("Weekday :" + index + " and Day is Thursday");
+			break;
+		case 5:
+			System.out.println("Weekday :" + index + " and Day is Friday");
+			break;
+		default:
+			System.out.println("Index is :" + index + " Invalid index");
+		}
+		return index;
+
+	}
+
+	public static void main(String[] args) {
+		Assignment69 ass = new Assignment69();
+		Scanner sc = new Scanner(System.in);
+		while (true) {
```

Comment:
Don't use scanner class unnecessary,  if it is asking in requirement then only use it